src/main/java/seedu/address/model/person/JobTitle.java

```diff
@@ -9,13 +9,15 @@
  */
 public class JobTitle {
 
-    public static final String MESSAGE_CONSTRAINTS = "Job titles can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS =
+            "For Job Titles (assuming job titles can only start with " +
+                    "alphabets and contain only alphabets and certain special characters from thereon)";
 
     /*
      * The first character of the job title must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
-    public static final String VALIDATION_REGEX = "[^\\s].*";
+    public static final String VALIDATION_REGEX = "^[A-Za-z][\\sA-Za-z_()\\[\\]<>@.#&+-]*$";
```

Comment:
I think there are much simpler patterns we can use for validation, depending on the desired effect, like \p{Alnum} represents all alphanumeric characters or \p{ASCII} represents all ASCII characters.

I don't really understand what this validation pattern accomplishes that [^\\s].* does not accomplish.

Reply:
You are right. Now that I think of it, I don't need think there is a real need to limit what users can input for their jobtitle. If they want to input nonsence its up to them I guess